src/main/java/goodgenerator/blocks/tileEntity/FuelRefineFactory.java

```diff
@@ -217,8 +217,13 @@ protected CheckRecipeResult validateRecipe(@NotNull GT_Recipe recipe) {
             @Override
             protected GT_OverclockCalculator createOverclockCalculator(@NotNull GT_Recipe recipe,
                     @NotNull GT_ParallelHelper helper) {
-                return super.createOverclockCalculator(recipe, helper)
-                        .setSpeedBoost(1F / (float) (1 << (Tier - recipe.mSpecialValue)));
+                long consumedPower = (long) recipe.mEUt * (1L << (Tier - recipe.mSpecialValue));
```

Comment:
`(long) recipe.mEUt << (Tier - recipe.mSpecialValue)` should work

Reply:
I just went with extracting the factor now.